_gsocproposals/2021/proposal_cling_dev.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Improve Cling’s Development Lifecycle
```

Comment:
There is already a proposal having this name, see: https://hepsoftwarefoundation.org/gsoc/2021/proposal_cling_cuda.html

Reply:
Doh, I copy-pasted the wrong title from the google doc. Good catch!!

Reply:
@agheata should I add a fix in this PR for another proposal? (here the title is correct)

Reply:
In the other proposal pls, we merge this after the fix

Reply:
Please check #930